synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -287,14 +264,14 @@ StateBone_Context::load_settings()
 		}
 
 		if(settings.get_value("bone.layer_skel",value) && value=="0")
-			set_layer_skel_flag(false);
+			layer_skel_togglebutton.set_active(false);
 		else
-			set_layer_skel_flag(true);
+			layer_skel_togglebutton.set_active(true);
```

Comment:
Instead of if-else why not just convert string `value` to int directly:
```c++
layer_skel_togglebutton.set_active(std::stoi(value));
```

Reply:
That way you call `set_active()` only once ;)

Reply:
Same for deform_skel toggle button too

Reply:
`load_settings()` is used both for initializing and loading of users' settings.
If we are on the very first run of Synfig Studio this value still doesn't exist and I dont't think that will work.